@here/harp-mapview/lib/geometry/TileGeometryCreator.ts

```diff
@@ -729,24 +729,29 @@ export class TileGeometryCreator {
                     object.onBeforeRender = chainCallbacks(
                         object.onBeforeRender,
                         (_renderer, _scene, _camera, _geometry, _material) => {
+                            const worldOffsetX =
```

Comment:
Can be moved inside Planar projection if statement

Reply:
Good idea, done.

Reply:
2e71b584c8bcd1314bf4c9ade4b83fdf23505ff5